test/two-commands.txt

```diff
@@ -0,0 +1,11 @@
+;;; TOOL: wat2wasm
+;;; FLAGS: -v
```

Comment:
llvm's lit uses "RUN" for this so maybe we should mirror that?  Could/should be separate change I guess.

Reply:
Eventually it might make sense to have flags be part of RUN too, no need for a separate line is there?

Reply:
Yeah, I want to do that too (move FLAGS into RUN)